deploy.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+
+NGINX_CONFIG=/etc/nginx/sites-available/adi-website
+ADI_WWWW=/srv/adi-website/www
+DOCKERFILE=Dockerfile
+PORT1=7000
+PORT2=6000
+NAME1="adi-website-1"
+NAME2="adi-website-2"
+
+
+if [ ! -e "$NGINX_CONFIG" ]; then
+    echo "Cannot find file $NGINX_CONFIG. Are you running this on the server?"
+    exit
+fi
+
+# Swap the port in nginx and in the Dockerfile
+PORT="$(cat $NGINX_CONFIG | grep 'set \$ADI_WEBSITE_PORT' | grep -o '[0-9]*')"
+if [ $PORT -eq $PORT1 ]; then
+    sed -i "s/$PORT1/$PORT2/g" $NGINX_CONFIG
+    sed -i "s/$PORT1/$PORT2/g" $DOCKERFILE
+elif [ $PORT -eq $PORT2 ]; then
+    sed -i "s/$PORT2/$PORT1/g" $NGINX_CONFIG
+    sed -i "s/$PORT2/$PORT1/g" $DOCKERFILE
+else
+    echo "adi-website running on unknown port: $PORT"
+    exit
+fi
+
+
+# Determine the new and old container names
+NEW_NAME=""
+OLD_NAME=""
+CURRENT_NAME="$(docker ps -a | grep adi-website | awk 'NF>1{print $NF}')"
```

Comment:
even better, just use `docker inspect`

```bash
CURRENT_NAME = $(docker inspect --format '{{ .Id }}' $NAME1) + $(docker inspect --format '{{ .Id }}' $NAME2)
```

Or something like that. I think it's better to use the CLI as opposed to grep + awk